zerver/views/thumbnail.py

```diff
@@ -39,7 +40,14 @@ def backend_serve_thumbnail(
         assert isinstance(maybe_user_profile, UserProfile)
         realm = maybe_user_profile.realm
 
-    if not validate_thumbnail_request(realm, maybe_user_profile, url):
+    is_authorized = validate_thumbnail_request(realm, maybe_user_profile, url)
+    if is_authorized is None:
+        if get_best_match(
+            request.META.get("HTTP_ACCEPT", ""),
+            ["image/jpeg", "image/png", "image/webp", "image/avif", "image/svg+xml"],
```

Comment:
The theory behind this is that we, the server, have a couple different options for content-types, and we want to find out which one the client prefers.  We don't have an `image/jpeg`, an `image/png`, and `image/webp`, etc.  We have a `image/png` and a `text/html` -- and we want to know which to send.

So there's no reason to send this laundry list of image types.  It doesn't matter what the client thinks about `image/jpeg` files -- we just want to compare the two types we might be sending.